notebooks/272-langchain-rag/272-langchain-rag.ipynb

```diff
@@ -0,0 +1,869 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    input_info = [("input_ids", input_shape, np.int64),
</pre><p>input info is not needed</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1512/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>